### DISCUSSION TITLE
Moving the collections tab to left side

### DISCUSSION THREAD

**Discussion by theabhayprajapati** [hoppscotch/hoppscotch#2350] [General]
how can place this to my left?

I want to send my collections tab to the  left side from right as shown in the home [@readme](https://github.com/hoppscotch/hoppscotch#--------------hoppscotch----------)

![image](https://user-images.githubusercontent.com/88815641/169469531-01b578a8-2804-47cf-aa51-c6a0a70ea280.png)

![image](https://user-images.githubusercontent.com/88815641/169469570-cb5ebe80-46b4-4fdf-a01f-ecc098d4bee1.png)

**Comment by AndrewBastin** [comment]
The toggle is available in `Settings > Experiments > Sidebar on Left`.
